### DISCUSSION TITLE
Geo Location Based Bandwidth Allocation

### DISCUSSION THREAD

**Discussion by MalithHatananchchige** [filecoin-project/saturn-node#371] [Q&A]
Hi, I understand the requirement why you guys need a 10Gbps uplink. But in some regions of the world, the average internet is very less. From where I'm coming from(Srilanka) the average speeds are 20Mbps and India is currently at 50Mbps. So if I'm running a node in Colombo Srilanka will the requirement check still look for 1300Mbps min bandwidth?

**Comment by DiegoRBaquero** [comment]
Yes, while the average speed is lower, a node must be able to serve multiple clients at the same time.
